src/components/QuickOpenModal.js

```diff
@@ -286,6 +292,51 @@ export class QuickOpenModal extends Component<Props, State> {
     return results && results.length ? results.length : 0;
   };
 
+  fuzzyhighlight = (query: any, results: ?Array<QuickOpenResult>) => {
+    const fullRes = [];
+
+    if (results) {
+      query = query.toLowerCase();
+      const queryUpper = query.toUpperCase();
+
+      let resultElem = 0;
+
+      for (resultElem; resultElem < results.length; resultElem++) {
+        let queryElem = 0;
+        if (query[0] === "@" || query[0] === "#" || query[0] === ":") {
+          queryElem++;
+        }
+
+        let titleElem = 0;
+        const title = results[resultElem].title;
+        const styledResults: Array<any> = [];
```

Comment:
Any chance to avoid any here? :)

Reply:
Currently no, because I have the highlighted element that has to be some JSX/HTML that render as Object object if I put in something like a string for example.